dhcp.nix

```diff
@@ -3,11 +3,81 @@ let
   cfg = config.pi-router;
 in
 {
+  systemd.network.enable = true;
 
-  # TODO Setup DHCP. Maybe KEA?
+  networking = {
+    useDHCP = false;
+    nameservers = cfg.dnsServers;
```

Comment:
This does nothing.
Our local Unbound server is being used (see `nslookup google.de`)
-> this only would apply when Unbound is disabled.
Maybe we should make this more clear?

Reply:
Removed.